### PR TITLE
feat(windows): support the new c++ react native for windows implement…

### PR DESCRIPTION
**Pull request by Daniel775**
…ation

# Overview

Adding  support for the new c++ react native windows implementations and take advantage of autolinking

# Test Plan

It supports the "open" method with following options: title, message and url.

teste

### CODE REVIEW

**Comment by Daniel775** [comment]
I believe everything is ok right now, let me know if there is any improvement required before approval

**Review by MateusAndrade — APPROVED** [review]

**Comment by MateusAndrade** [comment]
> I believe everything is ok right now, let me know if there is any improvement required before approval

Done! Forgot to merge this, sorry! 😅

**Pull request merged**
